src/pages/Experiment.js

```diff
@@ -0,0 +1,216 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { push } from 'redux-little-router';
+import { Modal, Button } from 'react-bootstrap';
+import PageHeader from '../components/PageHeader';
+import Loader from '../components/Loader';
+import Progress from '../components/Experiment/Progress';
+import Diff from '../components/Experiment/Diff';
+import Footer from '../components/Experiment/Footer';
+import {
+  load,
+  markCurrent,
+  ANSWER_SIMILAR,
+  ANSWER_MAYBE,
+  ANSWER_DIFFERENT,
+  ANSWER_SKIP,
+  experimentId,
+  getProgressPercent,
+  getCurrentFilePair,
+} from '../state/experiment';
+import { makeUrl } from '../state/routes';
+import './Experiment.less';
+
+const ALWAYS_ALLOW_SKIP_KEY = 'ca/ALWAYS_ALLOW_SKIP';
+
+class Experiment extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showModal: !!window.localStorage.getItem(ALWAYS_ALLOW_SKIP_KEY),
+      alwaysAllowSkip: false,
+    };
+    this.showModal = this.showModal.bind(this);
+    this.hideModal = this.hideModal.bind(this);
+    this.onAllowSkipChange = this.onAllowSkipChange.bind(this);
+  }
+
+  showModal() {
+    this.setState({ showModal: true });
+  }
+
+  hideModal() {
+    this.setState({ showModal: false });
+  }
+
+  onAllowSkipChange(e) {
+    this.setState({ alwaysAllowSkip: e.target.checked });
+    if (e.target.checked) {
+      window.localStorage.setItem(ALWAYS_ALLOW_SKIP_KEY, e.target.checked);
+    } else {
+      window.localStorage.removeItem(ALWAYS_ALLOW_SKIP_KEY);
+    }
+  }
+
+  render() {
+    const { user } = this.props;
+
+    return (
+      <div className="ex-page">
+        <PageHeader {...user} />
```

Comment:
again. Up to Ricardo. Also if you make your browser smaller it will be much closer to borders.

Reply:
But we agreed on try to follow the wireframe, didn't we?

Reply:
wireframes don't show how the page should look at different resolutions.

Reply:
So, if we agree that wireframes do not reflect the corner cases, we could fairly assume that title/paginator and avatar/progress/skip are pulled to the screen borders, couldn't we?

Reply:
If there is no specification it's up to developer for initial implementation. Then it gets validated by Ricardo. Sorry, I'm not going to change anything just because another developer has different "vision". We have very solid process for design: https://github.com/src-d/guide/pull/100 I'm going to follow it to archive the best UI experience.